### PR TITLE
ansible: add nim-waku to boot nodes

### PR DESCRIPTION
**Pull request by yakimant**
* https://github.com/status-im/infra-shards/issues/2

### CODE REVIEW

File: ansible/group_vars/boot.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+# Tag dependent on fleet: test
+nim_waku_cont_tag: 'deploy-shards-{{ stage }}'
+nim_waku_cont_name: 'nim-waku-boot'
+# TODO: switch for traces foe 1 node in host_vars
+nim_waku_log_level: 'debug'
+nim_waku_protocols_enabled: ['relay', 'filter', 'lightpush', 'peer-exchange']
+nim_waku_disc_v5_enabled: true
+nim_waku_dns4_domain_name: '{{ dns_entry }}'
+#nim_waku_node_key: '{{lookup("bitwarden", "fleets/shards/"+stage+"/nodekeys", field=hostname)}}'
```
Comment (jakubgs) [inline ansible/group_vars/boot.yml:12]: We'll need node keys won't we?

File: ansible/group_vars/boot.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+nim_waku_consul_success_before_passing: 5
+nim_waku_consul_failures_before_warning: 2
+nim_waku_consul_failures_before_critical: 20
+
+# Peer connecting
+waku_peers_rpc_port: '{{ nim_waku_rpc_tcp_port }}'
+waku_peers_rpc_timeout: 20
+waku_peers_rpc_retries: 5
+waku_peers_consul_services:
+  - { name: '{{ nim_waku_cont_name }}', env: '{{ env }}',  stage: '{{ stage }}' }
```
Comment (jakubgs) [inline ansible/group_vars/boot.yml:57]: We don't need this whole section, since we're probably not going to be using this role.

**Review by jakubgs — APPROVED** [review]
Mostly ready.

File: ansible/group_vars/boot.yml (hunk 2 above)
Comment (yakimant) [reply ansible/group_vars/boot.yml]: Done!

File: ansible/group_vars/boot.yml (hunk 1 above)
Comment (yakimant) [reply ansible/group_vars/boot.yml]: Done, copied the local keys to bitwarden

**Pull request merged**